app/facades/items_facade.rb

```diff
@@ -0,0 +1,11 @@
+class ItemsFacade
+  class << self
+
+    def get_all_items
+      items = ItemService.all_items
+      items.each do |item|
```

Comment:
I think this `.each` is causing the bug you described in your submission.
Because you're using `each` here, and `map` in `MerchantsFacade`, the `MerchantsFacade` returns Merchant POROs while this method still returns the original array (a hash of Item data).